### ISSUE TITLE
error reporting

### ISSUE THREAD

**Issue by missinglink** [pelias/admin-lookup#8]
I think there is an error with either the lib or my setup but I don't get any errors, any idea what's going on here?

```bash
$ mkdir /tmp/sandbox && cd !$;
$ npm install pelias-admin-lookup;
$ node -e "require('pelias-admin-lookup').stream(console.log.bind(console, 'loaded'));"
$ echo $?
0
```

```bash
pelias-admin-lookup@1.0.2 ../node_modules/pelias-admin-lookup
```

@sevko

**Comment by missinglink** [comment]
```bash
$ pelias-config 
{
  "esclient": {
    "apiVersion": "1.1",
    "keepAlive": true,
    "requestTimeout": "120000",
    "hosts": [
      {
        "env": "development",
        "protocol": "http",
        "host": "localhost",
        "port": 9200
      }
    ],
    "log": [
      {
        "type": "stdio",
        "level": [
          "error",
          "warning"
        ]
      }
    ]
  },
  "imports": {
    "geonames": {
      "datapath": "./data"
    },
    "quattroshapes": {
      "datapath": "/media/hdd/simplified"
    },
    "openstreetmap": {
      "datapath": "/media/hdd/mapzen-metro",
      "leveldbpath": "/media/hdd/pelias-tmp",
      "import": [
        {
          "type": {
            "node": "osmnode",
            "way": "osmway"
          },
          "filename": "wellington_new-zealand.osm.pbf"
        }
      ]
    }
  },
  "elasticsearch": {
    "settings": {
      "index": {
        "refresh_interval": "-1"
      }
    }
  },
  "logger": {
    "level": "debug"
  }
}
```

```bash
$ l /media/hdd/simplified
total 2.1G
drwxrwxr-x  2 peter         peter         4.0K Nov  6 11:01 .
drwxrwxrwx 20 elasticsearch elasticsearch 4.0K Feb 26 16:55 ..
-rw-rw-r--  1 peter         peter         388K Nov  5 20:17 qs_adm0.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:17 qs_adm0.prj
-rw-rw-r--  1 peter         peter          76M Nov  5 20:17 qs_adm0.shp
-rw-rw-r--  1 peter         peter         2.7K Nov  5 20:17 qs_adm0.shx
-rw-rw-r--  1 peter         peter          11M Nov  5 20:17 qs_adm1.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:17 qs_adm1.prj
-rw-rw-r--  1 peter         peter          69M Nov  5 20:17 qs_adm1.shp
-rw-rw-r--  1 peter         peter          37K Nov  5 20:17 qs_adm1.shx
-rw-rw-r--  1 peter         peter          63M Nov  5 20:17 qs_adm2.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:17 qs_adm2.prj
-rw-rw-r--  1 peter         peter         143M Nov  5 20:17 qs_adm2.shp
-rw-rw-r--  1 peter         peter         143K Nov  5 20:17 qs_adm2.shx
-rw-rw-r--  1 peter         peter         441M Nov  5 20:18 qs_localadmin.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:18 qs_localadmin.prj
-rw-rw-r--  1 peter         peter         211M Nov  5 20:18 qs_localadmin.shp
-rw-rw-r--  1 peter         peter         861K Nov  5 20:18 qs_localadmin.shx
-rw-rw-r--  1 peter         peter         783M Nov  5 20:18 qs_localities.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:18 qs_localities.prj
-rw-rw-r--  1 peter         peter         188M Nov  5 20:18 qs_localities.shp
-rw-rw-r--  1 peter         peter         1.3M Nov  5 20:18 qs_localities.shx
-rw-rw-r--  1 peter         peter         123M Nov  5 20:18 qs_neighborhoods.dbf
-rw-rw-r--  1 peter         peter          256 Nov  5 20:18 qs_neighborhoods.prj
-rw-rw-r--  1 peter         peter          40M Nov  5 20:18 qs_neighborhoods.shp
-rw-rw-r--  1 peter         peter         390K Nov  5 20:18 qs_neighborhoods.shx
```

**Comment by sevko** [comment]
Must be [this change](https://github.com/pelias/admin-lookup/pull/7), implemented as per #6 . It's no longer an asynchronous interface on the surface, and `.stream()` accepts no callbacks. See the PR description/updated README for a thorough description.
